app/src/main/sqldelight/ca/llamabagel/transpo/data/db/RecentRoute.sq

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE recent_route (
+    id TEXT NOT NULL UNIQUE PRIMARY KEY,
+    short_name TEXT NOT NULL,
+    long_name TEXT,
+    type TEXT NOT NULL,
+    date INTEGER NOT NULL,
+    FOREIGN KEY (id) REFERENCES route_fts(id) ON DELETE CASCADE
+);
+
+insert:
+REPLACE INTO recent_route(id, short_name, long_name, type, date)
+VALUES(:id, :shortName, :longName, :type, :date);
+
+selectMostRecent:
```

Comment:
use `get` terminology instead of `select`. The function will be consumed in Kotlin, not SQL.
```suggestion
getMostRecent:
```